lightning/src/offers/offer.rs

```diff
@@ -358,77 +358,86 @@ pub(super) struct OfferContents {
 	signing_pubkey: PublicKey,
 }
 
-impl Offer {
+macro_rules! offer_accessors { ($self: ident, $contents: expr) => {
 	// TODO: Return a slice once ChainHash has constants.
 	// - https://github.com/rust-bitcoin/rust-bitcoin/pull/1283
 	// - https://github.com/rust-bitcoin/rust-bitcoin/pull/1286
```

Comment:
These are merged now, could add the TODO to #1970

Reply:
Looks like we need to update to 0.30 first.